src/components/CompaniesGrid/index.tsx

```diff
@@ -0,0 +1,29 @@
+import s from "./style.module.scss";
+
+const companies = [
+  {
+    name: "Merck",
+    img: "asset/about-star.png",
```

Comment:
Since this is going on the live website, we can't publish this until all of the data here is real and correct. That includes the company names, images, and hyperlinks. If we need to update more data later, we can continue updating the code.